justfile

```diff
@@ -64,6 +64,24 @@ check_v2 *FLAGS:
     cargo check {{ check_flags }} --no-default-features --features "${FEATURES}" -- {{ FLAGS }}
     set +x
 
+build_v2 *FLAGS:
+    #! /usr/bin/env bash
+    set -euo pipefail
+
+    FEATURES="$(cargo metadata --all-features --format-version 1 --no-deps | \
+        jq -r '
+            [ .packages[] | select(.name == "router") | .features | keys[] # Obtain features of `router` package
+            | select( any( . ; test("(([a-z_]+)_)?v2") ) ) ] # Select v2 features
+            | join(",") # Construct a comma-separated string of features for passing to `cargo`
+    ')"
+
+    set -x
+    cargo build --package router --bin router --no-default-features --features "${FEATURES}" {{ FLAGS }}
+    set +x
+
+alias bb := build_v2
```

Comment:
We haven't kept any aliases for the v2 related recipes so far...

Reply:
OK, I will remove he alias for now